project/Docker.scala

```diff
@@ -0,0 +1,29 @@
+import com.typesafe.sbt.packager.Keys._
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{Docker => docker}
+import sbt.Def
+import sbt.Keys._
+
+object Docker {
+
+  lazy val dockerSettings = Seq(
+    packageName in docker := packageName.value,
+    dockerBaseImage := "anapsix/alpine-java:8",
```

Comment:
I think explicit is probably a good idea (so the 8 would be replaced by the minor release). 

On a side note what do people think of [this](http://blog.takipi.com/running-java-on-docker-youre-breaking-the-law/)? We might need to use a base image that uses OpenJDK...